docs/source/en/quantization/torchao.md

```diff
@@ -22,6 +22,8 @@ pip install --upgrade torch torchao transformers
 
 By default, the weights are loaded in full precision (torch.float32) regardless of the actual data type the weights are stored in such as torch.float16. Set `torch_dtype="auto"` to load the weights in the data type defined in a model's `config.json` file to automatically load the most memory-optimal data type.
 
+If you want to run the following codes on CPU even with GPU available, just change `device_map="cpu"` and `quantization_config = TorchAoConfig("int4_weight_only", group_size=128, layout=Int4CPULayout())` where `layout` comes from `from torchao.dtypes import Int4CPULayout` which is only available from torchao 0.8.0 and higher.
```

Comment:
I'm wondering if we are exposing too much details here, we are also going through a refactor of quantize_ API to change it to take configs, maybe we can hide `Int4CPULayout` as well

Reply:
I agree! Passing `layout` is not user-friendly. Once your refactor is complete, I will update this document. Thanks!